core/src/banking_stage.rs

```diff
@@ -1069,8 +1071,11 @@ mod tests {
             let blocktree = Arc::new(
                 Blocktree::open(&ledger_path).expect("Expected to be able to open database ledger"),
             );
+            let mut poh_config = PohConfig::default();
+            // limit the tick to 1 to prevent clearing working_bank at PohRecord then PohRecorderError(MaxHeightReached) at BankingStage
+            poh_config.target_tick_count = Some(1);
```

Comment:
Instead of the magic number 1, can we use something like genesis_block.ticks_per_slot - 1, to make it clear that there's nothing special about this number, it just needs to be < genesis_block.ticks_per_slot

Reply:
sure thing! done at https://github.com/solana-labs/solana/pull/6251/commits/a9138d1efdcaff8d1b8979c638c13aefab0ed444

Reply:
Well, I couldn't directly use genesis_block.ticks_per_slot because subtracting -1 from it still causes the race condition, so I chose different one: b07bf58e0cfc580f65e08ee731ce88fc2af8669e